schemas/com.snowplowanalytics.snowplow.media/ad_break_end_event/jsonschema/1-0-0

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Media player event that signals the end of an ad break.",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow.media",
+    "name": "ad_break_end_event",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+  },
```

Comment:
Out of curiosity, does this avoid creating a column/table for the destinations that do that? Or do you just get like an empty struct? This is really nice if it just populates `event_name` & co without any other footprint.

Reply:
I actually don't know tbh, but I would assume that it no column/table is created. We do have a few other common events without parameters that we track for instance the `application_install`, foreground and background event in the mobile trackers.

Reply:
Looked through the dev1 pipeline tables for `application_install`:

- BigQuery: nullable string always containing `{}`
- Snowflake: normal `object` like any other unstruct, but with empty definition so essentially `{}`
- Redshift: still creates a table, table just has the boilerplate fields with no other columns

So not quite as ergonomic as I'd hoped, and kind of wasteful to boot. :disappointed:
Now morbidly curious what happens with `"type": "null"` rather than empty object. :trollface:

Reply:
That's a bit sad, but good to know!